WordPress/gutenberg
# Add command for publish post types

## What?
See https://github.com/WordPress/gutenberg/issues/50407

## Why?
Currently publish post command is not available.

## How?
Added command for publishing  post type  from command box.

## Testing Instructions

- Go to Posts  in the WordPress admin.
- Open a new or existing post to enter the block editor.
- In the block editor, open the Command Palette (by pressing Cmd + K on Mac).
- Check for the publish post command.
- Select the command and confirm it  publish post.
- If the post is not already in draft, click Save as Draft or Preview to set it to a non-published state.
- Then again check for the publish post command


## Screenshots or screencast


https://github.com/user-attachments/assets/e30b9022-a92b-4352-ba44-8fb9ccb912bf
